src/electron/electron/main/services/WindowService.ts

```diff
@@ -265,131 +268,170 @@ export class WindowService {
+    const trayImage = nativeImage.createFromPath(appIcon)
+    trayImage.setTemplateImage(true)
+    this.tray = new Tray(trayImage)
+    await this.updateTray()
   }
 
-  private getTrayMenuTemplate(): MenuItemConstructorOptions[] {
-    const versionAndUpdate: MenuItemConstructorOptions[] = [
+  private async getTrayMenuTemplate(): Promise<MenuItemConstructorOptions[]> {
+    const settings: Settings = await Settings.findOne({ where: { onlyOneEntityShouldExist: 1 } })
+    const trayMenuItems: MenuItemConstructorOptions[] = [
       { label: `Version ${app.getVersion()}`, enabled: false },
       {
         label: 'Check for updates',
         enabled: false,
         click: () => this.appUpdaterService.checkForUpdates({ silent: false })
       },
-      { type: 'separator' }
-    ];
-    const windowMenu: MenuItemConstructorOptions[] = [
-      {
-        label: 'Open Onboarding',
-        visible: is.dev,
-        click: async () => {
-          const shouldShowStudyTrackersStarted = !!(await Settings.findOneBy({
-            studyAndTrackersStartedShown: false,
-            onboardingShown: true
-          }));
-          await this.createOnboardingWindow(
-            shouldShowStudyTrackersStarted ? 'study-trackers-started' : undefined
-          );
-        }
-      }
-    ];
-    const otherMenu: MenuItemConstructorOptions[] = [
+      { type: 'separator' },
       {
-        label: 'About',
-        click: () => this.createAboutWindow()
+        label: `Subject ID: ${settings.subjectId}`,
+        enabled: false,
       },
       {
-        label: 'Get Help',
-        click: (): void => {
-          const mailToAddress = studyConfig.contactEmail;
-          shell.openExternal(`mailto:${mailToAddress}`);
-        }
+        label: 'Copy Subject Id',
+        click: () => clipboard.writeText(settings.subjectId)
       },
       {
-        label: 'Report a Problem',
-        click: (): void => {
-          const mailToAddress = studyConfig.contactEmail;
-          shell.openExternal(`mailto:${mailToAddress}`);
-        }
+        label: `Days participated: ${settings.daysParticipated}`,
+        enabled: false,
+        visible: studyConfig.displayDaysParticipated
       },
       { type: 'separator' },
       {
-        label: 'Open Logs', // todo: move to settings
-        click: (): void => {
-          LOG.info(`Opening logs at ${app.getPath('logs')}`);
-          shell.openPath(`${app.getPath('logs')}`);
-        }
+        label: 'Open Experience Sampling',
+        click: () => this.createExperienceSamplingWindow(true)
       },
       {
-        label: 'Open Collected Data', // todo: move to settings
-        click: (): void => {
-          LOG.info(`Opening collected data at ${app.getPath('userData')}`);
-          shell.showItemInFolder(path.join(app.getPath('userData'), 'database.sqlite'));
+        label: 'Open Task Planning',
+        visible: studyConfig.trackers.taskTracker?.enabled,
```

Comment:
just to be sure as it's currently showing up for me: is taskTracker enabled for this test?

(i don't see it in study.config.ts)

Reply:
sorry, my bad. Did not realize that undefined would default to "true" in this case since it's an optional property. Fixed now.